internal/builders/gradle/create_attestation.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# "version" and "attestations" fields:
+echo -e -n "{\n  \"version\": 1,\n  \"attestations\": [" >> "$SLSA_OUTPUTS_ARTIFACTS_FILE"
+
+num_jar_files=$(find ./release-files-for-slsa -type f | wc -l)
+counter=1
+        
+# Add one attestation per .jar file:
+find ./release-files-for-slsa -type f -print0 | while read -r -d $'\0' fname
+do
+          
+    bn=$(basename -- "$fname")
+    hash=$(sha256sum "$fname" | awk '{print $1}')
+          
+    echo -n "
+        {
+          \"name\": \"${bn}.intoto\",
+          \"subjects\": [
+            { \"name\": \"${bn}\",
+              \"digest\": { \"sha256\": \"${hash}\"  }
+            }
+          ]
+        }" >> "$SLSA_OUTPUTS_ARTIFACTS_FILE"
+          
+    # Add comma between attestations and not after the last
+    if [[ "$COUNTER" != "$NUM_JAR_FILES" ]]; then
+      echo -n "," >> "$SLSA_OUTPUTS_ARTIFACTS_FILE"
+    fi
+         
+    COUNTER=$[$COUNTER +1]
```

Comment:
You'll need to update these references as well
```suggestion
    if [[ "$counter" != "$num_jar_files" ]]; then
      echo -n "," >> "$SLSA_OUTPUTS_ARTIFACTS_FILE"
    fi
         
    counter=$[$counter +1]
```